oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -296,7 +296,9 @@ Trigger events for removing a CC would be:
 
 * A CC informs the Program Administrators that they can no longer continue in
   the role
-* A CC has been inactive and unreachable for more than 6 months
+* A CC has not been able to fulfill the commitments made in the
+  `Revised Declaration of Commitment to the Core Contributor Program`_,
+  or has been unreachable, for more than 6 months
```

Comment:
Is it possible in the situation where a CC can't meet their commitments to update the level of commitment they agree to in their declaration?  Is that already documented somewhere and can be linked here? Or can we add something to that effect?  I see that the level of commitment varies across CCs and CC orgs so perhaps if they are not meeting their commitment, updating their level of commitment to better match the work they are actually doing is a good alternative to fully removing them from the program?

This is not a blocker in my mind but perhaps an alternative worth clarifying?  I think given that the result is a discussion with the program administrator, we already have the flexibility to enact the above option but perhaps it's worth mentioning explicitly?

Reply:
I also think we should be a little careful with this, because hypothetically a community member (not joining on behalf of an organization) could get involved, and they wouldn't be necessarily a signatory to the Declaration.

Could we incorporate something like 

```
CCs may `Contact the Program Administrators`_ if they are struggling
with their ability to fulfill their commitment.
```
?

Reply:
hm, ok I see you do this in the next section.

@feanil I think what you're asking is good, but nuanced and maybe too much/too complicated to put into the OEP. Suggesting that the CC get in touch with the Program Administrators when things are going south seems like the best first step to take.

Reply:
@feanil @sarina 

> Is it possible in the situation where a CC can't meet their commitments to update the level of commitment they agree to in their declaration?

Yes this kind of measures would definitely be good to consider when this happens -- the intent is what you describe @sarina , to start by discussing, and letting the administrator arbitrate it individually. If some patterns emerge, we can always discuss them then, but this way we get to go through a few cases before?

Reply:
@sarina Yup - the important bit is to establish the principle, and have it clearly acknowledged by everyone. But we keep the latitude to adapt how it is implemented to individual cases. Revisiting once we face actual problems and can have a discussion is actually the goal, so we can be practical when figuring out solutions.

Reply:
Great, got it. I can see not wanting to caveat what to do if you're finding yourself falling behind in this section; you mention it later and that's a better place. However, I wonder if it's worth linking to like:

```
A CC has not been able to fulfill the commitments made in the
  `Revised Declaration of Commitment to the Core Contributor Program`_,
  or has been unreachable, for more than 6 months. See `fulfilling CC commitments`_
```

with a `.. fulfilling CC commitments` line at 309.

I'm not wedded to that wording, pick what you would like if you take this suggestion.

Reply:
@sarina Sounds good! Done.